public/manifest.json

```diff
@@ -1,14 +1,14 @@
 {
-  "short_name": "app-prototype",
-  "name": "app-prototype",
+  "short_name": "React App",
+  "name": "Create React App Sample",
```

Comment:
I think these need to change back to `app-prototype`. That way they should get replaced with the generated app name when someone runs the `npx spraygun` command.

Reply:
ah good catch thanks